vscode-extension/src/extension.ts

```diff
@@ -177,6 +187,36 @@ async function createNewAIConfig(
   );
 }
 
+/**
+ *
+ */
+async function openConfigFile() {
+  let defaultUri;
+  const activeDocument = vscode.window.activeTextEditor?.document;
+
+  if (activeDocument && isSupportedConfigExtension(activeDocument.fileName)) {
+    defaultUri = activeDocument.uri;
+  }
+
+  const openUri = await vscode.window.showOpenDialog({
+    defaultUri,
+    filters: {
+      // Allow opening .json and .yaml files regardless of .aiconfig. sub-extension
+      "AIConfig Extension": SUPPORTED_FILE_EXTENSIONS,
+    },
+  });
```

Comment:
I'm wondering if we should update this command or add a separate command that does `Open Active File in AIConfig Editor` (wording TBD), which instead of opening the file dialog just opens the current active file as an aiconfig.

Reason being there could be untitled files (e.g. copy-pasting content into a text editor) and wanting to open that as an aiconfig.

Reply:
I think it might make sense to have as a separate command. Otherwise, if it's part of this command we need to surface that choice somehow (e.g. like a dialog w/ button "current" or "search") and that makes it an extra click to get through this (imo way more common) flow.

Reply:
Going to land this and can follow up with the other command tomorrow